src/main/scala/mesosphere/marathon/state/AppDefinition.scala

```diff
@@ -288,7 +288,7 @@ case class AppDefinition(
     }
 
     AppDefinition(
-      id = PathId(proto.getId),
+      id = AbsolutePathId(proto.getId), // TODO AN: Are we sure that we have always stored an absolute path id in the past?
```

Comment:
Maybe a Marathon archeologist will be able to answer this question /cc @meichstedt

Reply:
I'm pretty sure we have, but I better double check that.

Reply:
@ANeumann82 I'm pretty sure we have as well, also.